guides/network/using-middleware.md

```diff
@@ -1,52 +0,0 @@
-# Using middleware
```

Comment:
Maybe instead of deleting the file, put here a link to the new file? This will avoid broken links in issues, etc.